src/content/docs/zh-cn/guides/content-collections.mdx

```diff
@@ -1,7 +1,8 @@
 ---
 title: 内容集合
 description: >-
-  内容集合有助于组织你的 Markdown 文件以及通过模式使用带有类型检查的 frontmatter。
+  内容集合有助于组织你的 Markdown 文件以及通过 schema 使用带有类型检查的 frontmatter。
```

Comment:
Are you sure that keep "schema" without translation? If that, maybe you also need to add this word to terms of translation in [Chinese translation guide](https://github.com/withastro/docs/tree/main/src/i18n/zh-cn) in next PR. 🤔